Core xdr-retrieve-file different separator options (#45106)
demisto/content: Packs/Core/ReleaseNotes/3_5_79.md

#### Integrations

##### Investigation & Response

> - Added the *paths_separator* argument to the ***core-retrieve-files*** command, allowing retrieval of files whose paths contain a comma by specifying a different delimiter.

Reviewer: ```suggestion
- Added the ***paths_separator*** argument to the ***core-retrieve-files*** command to support custom delimiters for paths containing commas.
```
